src/assets/source/scripts/main.js

```diff
@@ -74,6 +75,9 @@ require.config({
         underscore: {
             exports: '_'
         },
+        d3: {
+            exports: 'd3'
```

Comment:
You're right, seems to be unnecessary. Used an older version (3.1.) before, where this was necessary. I removed it and it still works fine.